src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -99,4 +132,15 @@ Enable the DNS challenge for a domain managed on Cloudflare with account credent
 tls {
 	dns cloudflare {env.CLOUDFLARE_API_TOKEN}
 }
-```
\ No newline at end of file
+```
+
+Enable TLS Client Authentication
```

Comment:
Explain what kind of authentication, or what is necessary and sufficient to auth